docs/optimization-framework/ab-testing.md

```diff
@@ -35,20 +31,24 @@ It is enabled by default but can be disabled via the configuration file:
 }
 ```
 
-Within the post edit screen click on the A/B icon to access the Experiments panel. Here you can set the alternative titles, the amount of traffic to show the tests to as well as start and end dates.
+Within the post edit screen click on the A/B icon to access the Experiments panel. Here you can set the alternative titles and/or alternative featured images, the amount of traffic to show the tests to as well as start and end dates.
 
 Once you are ready to run the test click on the toggle to unpause it. Results are updated every hour until a statistically significant winner is found.
 
 ![A/B Testing Titles user interface](../assets/ab-tests-titles.png)
 
-By default post title A/B tests are enabled for Posts and Pages however custom post types can be supported using the `altis.experiments.titles` key either when registering the post type or in the `init` action, for example:
+By default post title and featured image A/B tests are enabled for Posts and Pages however custom post types can be supported using the `altis.experiments.titles` key either when registering the post type or in the `init` action, for example:
 
 ```php
 add_action( 'init', function () {
 	add_post_type_support( 'events', 'altis.experiments.titles' );
+	// and/or..
+	add_post_type_support( 'events', 'altis.experiments.featured_images' );
 } );
 ```
 
+Note that in order for the test to function correctly, it needs WordPress to output the `<ab-test></ab-test>` Web Component that includes all the variations of the feature representation, so the front-end would be able to switch to the proper variation to serve to the visitor. And such, the built-in features will only work if the native functions are used, which are `the_title` / `get_the_title` (for post title) and `the_post_thumbnail` / `get_the_post_thumbnail` (for featured image).
```

Comment:
```suggestion
Note that in order for the test to function correctly, it needs WordPress to output the `<ab-test></ab-test>` Web Component (generated by the `output_ab_test_html_for_post()` function) that includes all the variations of the feature representation, so the front-end would be able to switch to the proper variation to serve to the visitor. As such, the built-in features will only work if the native WordPress functions are used, which are `the_title()` or `get_the_title()` for post title and `the_post_thumbnail()` or `get_the_post_thumbnail()` for featured image.
```

Reply:
This also makes me think we should document the web components too like we have here currently for personalization https://docs.altis-dxp.com/v8/analytics/native/experience-blocks/#personalization

So something like "it needs WordPress to output the `<ab-test></ab-test>` or `<ab-test-block></ab-test-block>` Web Component". Then perhaps to link to a separate sub page or just document them inline.

For reference `<ab-test>` is designed for inline content and has no impact on the layout or styling of the site. `<ab-test-block>` is for block level content and will output the content in a wrapper with the default CSS `display: block`.

Reply:
I had started to document the `<ab-test/>` component with a headless use case, but stopped mid-way realizing that I don't really know how it works yet 😅 and that there is quite a bit to document rather than just the block. So I'd probably exclude that as a goal from the update here. A new issue would be a bit more efficient.